documentation/src/main/asciidoc/topics/search.adoc

```diff
@@ -0,0 +1,1087 @@
+[[indexing_searching]]
```

Comment:
Just looking at adding a link to the index.html page. Could we change this to `[id='search_api']` ? That can be our permanent anchor for top-level search docs.

Reply:
sure

Reply:
done